contracts/CTKNCrowdsale.sol

```diff
@@ -23,6 +26,7 @@ contract CTKNCrowdsale is CappedCrowdsale, RefundableCrowdsale, MintedCrowdsale
         uint256 _cap,
         uint256 _goal,
         address _wallet,
+        address _refundWallet,
```

Comment:
and maybe _overpaymentWallet.  Not sure what the best word is, perhaps its fine, just don't want to mix up the different types of refund.